packages/nimble-components/src/table/index.ts

```diff
@@ -1,6 +1,17 @@
+import { observable } from '@microsoft/fast-element';
 import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import {
+    ColumnDef,
+    TableState,
+    Updater,
+    Table as TanstackTable,
+    createTable,
+    getCoreRowModel,
+    TableOptionsResolved
+} from '@tanstack/table-core';
```

Comment:
@atmgrifter00 was mentioning that this may break blazor apps until the rollup update is in that builds `all-components.ts`. If that's true we could either remove the table from `all-components.ts` temporarily or we could fix that rollup build in the PR. Seems like we should make sure blazor does not break with this change.

Reply:
I talked with @atmgrifter00 about this, and he confirmed that Blazor does not break with this change. You just can't pull the nimble-table into a Blazor app until the rollup update is in. Given that, I'm going to resolve this conversation, and defer the work to get the rollup updated to a different PR.